lib/ansible/module_utils/vmware.py

```diff
@@ -234,11 +234,11 @@ def compile_folder_path_for_object(vobj):
     thisobj = vobj
     while hasattr(thisobj, 'parent'):
         thisobj = thisobj.parent
+        if thisobj.name == 'Datacenters':
```

Comment:
Doesn't look like that should be the intention... I guess it should be `continue`?